### PR TITLE
feat(exa): --no-icons

### PR DESCRIPTION
**Pull request by ben5vik**

### CODE REVIEW

File: src/exa.ts [hunk 1]
```diff
@@ -252,6 +252,10 @@ const completionSpec: Fig.Spec = {
       description: "Shows icons before each folder or file",
       name: "--icons",
     },
+    {
+      description: "Does not display icons (This overwrites -icons)",
+      name: "--no-icons", 
+    }
```
Comment (fedeci) [inline src/exa.ts:258]: These are mutually `exclusiveOn`, can you add that property please?

**Review by fedeci — CHANGES_REQUESTED** [review]

File: src/exa.ts [hunk 2]
```diff
@@ -252,6 +252,10 @@ const completionSpec: Fig.Spec = {
       description: "Shows icons before each folder or file",
       name: "--icons",
```
Comment (fedeci) [inline src/exa.ts:253]:
```suggestion
      name: "--icons",
      exclusiveOn: ["--no-icons"],
```

File: src/exa.ts [hunk 3]
```diff
@@ -252,6 +252,10 @@ const completionSpec: Fig.Spec = {
...
     },
+    {
+      description: "Does not display icons (This overwrites -icons)",
```
Comment (fedeci) [inline src/exa.ts:256]:
```suggestion
      description: "Does not display icons",
      exclusiveOn: ["--icons"],
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Sorry I should have pointed you in the right direction, this is the best way to do it since we can then only suggest an option when it is really relevant!

**Comment by ben5vik** [comment]
Ya, I also noticed the `--no-icons` is not that important because u could just not use `--icons` in order to also have `--no-icons`

**Review by fedeci — APPROVED** [review]
Thanks! It looks good to me!

File: src/exa.ts [hunk 4]
```diff
@@ -251,6 +251,12 @@ const completionSpec: Fig.Spec = {
     {
       description: "Shows icons before each folder or file",
       name: "--icons",
+      exclusiveOn: ["--no-icons"],
+    },
+    {
+      description: "Does not display icons",
+      exclusiveOn: ["--icons"],
+      name: "--no-icons",
```
Comment (fedeci) [inline src/exa.ts:259]:
```suggestion
      name: "--no-icons",
```

**Pull request merged**